libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -155,27 +172,26 @@ AC_WPNav::AC_WPNav(const AP_InertialNav& inav, const AP_AHRS_View& ahrs, AC_PosC
 /// init_loiter_target in cm from home
 void AC_WPNav::init_loiter_target(const Vector3f& position, bool reset_I)
```

Comment:
Aha, it seems the reset_I parameter doesn't do anything anymore.  We need to be careful here because PosHold is apparently the only place this was ever used.  @lthall?

Reply:
Lets take it out for now.

Reply:
I'd forgotten about this item but I've removed it just now from the new PR (using lthall-new-loiter13)